lib/src/input/figma/entities/layers/vector.dart

```diff
@@ -85,9 +76,29 @@ class FigmaVector extends FigmaNode implements FigmaNodeFactory {
   Map<String, dynamic> toJson() => _$FigmaVectorToJson(this);
 
   @override
-  Future<PBDLNode> interpretNode() {
+  Future<PBDLNode> interpretNode() async {
+    if (figmaStyleProperty.fills.first.type
```

Comment:
What would happen if there is a vector with multiple fills? 🤔 

Also, what if we had a vector like the following: 
![image](https://user-images.githubusercontent.com/42812006/155242527-b8d76e96-dd5c-4a79-b20c-a82a46b1df09.png)


Would we interpret this as a square container, since it would process it as a PBDLRectangle?